src/libs/ReportActionsUtils.ts

```diff
@@ -1808,6 +1808,13 @@ function getRemovedFromApprovalChainMessage(reportAction: OnyxEntry<ReportAction
     return translateLocal('workspaceActions.removedFromApprovalWorkflow', {submittersNames, count: submittersNames.length});
 }
 
+function getDemotedFromWorkspaceMessage(reportAction: OnyxEntry<ReportAction<typeof CONST.REPORT.ACTIONS.TYPE.DEMOTED_FROM_WORKSPACE>>) {
+    const originalMessage = getOriginalMessage(reportAction);
+    const policyName = originalMessage?.policyName ?? translateLocal('workspace.common.workspace');
+    const oldRole = translateLocal('workspace.common.roleName', {role: originalMessage?.oldRole}).toLowerCase();
```

Comment:
Why are we calling toLowerCase here?

Reply:
I wanted to localize the roles but also wanted to keep the copy exactly as requested (including case).

> [Admin] updated your role in [Workspace Name] from admin to user. You have been removed from all submitter workspace chats except for your own.

Reply:
Ah makes sense